pkg/trie/triedb/in_memory_to_triedb_migration_test.go

```diff
@@ -18,7 +19,45 @@ func newTestDB(t assert.TestingT) database.Table {
 	return database.NewTable(db, "trie")
 }
 
-func TestTrieDB_Migration(t *testing.T) {
+func TestWriteTrieDB_Migration(t *testing.T) {
+	inmemoryTrieDB := newTestDB(t)
+	inMemoryTrie := inmemory.NewEmptyTrie()
+	inMemoryTrie.SetVersion(trie.V1)
+
+	inmemoryDB := db.NewMemoryDB(make([]byte, 1))
+	trieDB := NewTrieDB(trie.EmptyHash, inmemoryDB, nil)
+
+	entries := map[string][]byte{
```

Comment:
Similar to below suggestion, I would say add something to make the values a bit different